Sources/MapboxMaps/Gestures/GestureRecognizers/MapboxLongPressGestureRecognizer.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import UIKit
+
+internal final class MapboxLongPressGestureRecognizer: UILongPressGestureRecognizer {
```

Comment:
Can you, please, add some documentation on how this custom gesture recognizer differs from the standard one.